README.md

```diff
@@ -2,4 +2,6 @@
 
 This repository contains internal REST backup server image dockerfiles for Eclipse Che.
 
+The image uses restic REST server that supposed to store Che backups.
+
 It is designed to be deployed by Che Operator by setting `autoconfigureRestBackupServer` to `true` in an instance of `CheClusterBackup` and be accecible only from within the cluster.
```

Comment:
```suggestion
```

Can remove this? Because if we change the field name we will have to update the README.
Instead of this I propose to document `autoconfigureRestBackupServer` field to mention which backup server will be used.

Reply:
ok